### ISSUE TITLE
completion (bash/zsh)

### ISSUE THREAD

**Issue by sonnerberg** [dbwebb-se/dbwebb-cli#7]
I just installed `dbwebb` and would love to see completion working.

I don't have the knowledge myself to implement completion for `bash`/`zsh` but I think it would be a handy feature.

I would love to make my own pull request if I ever figure it out but it might be a while.

Here is a [link](https://github.com/zsh-users/zsh-completions/blob/master/zsh-completions-howto.org) describing how to create completion scripts for `zsh` and I include it to make it clear what I am proposing.

**Comment by mosbth** [comment]
Yes, thats a handy feature. I looked into it earlier but have not yet implemented it. 
Its a good suggestion.
